crates/query-engine/translation/src/translation/query/native_queries.rs

```diff
@@ -33,7 +33,7 @@ pub fn translate(env: &Env, state: State) -> Result<Vec<sql::ast::CommonTableExp
                         None => Err(Error::ArgumentNotFound(param.clone())),
                         Some(argument) => match argument {
                             models::Argument::Literal { value } => {
-                                values::translate_json_value(value, &typ)
+                                values::translate_json_value(&mut State::new(), value, &typ)
```

Comment:
Please document why we're constructing a new state here.

Reply:
Done~